### PR TITLE
New package: wahibre.mtn version 3.4.2

### PR DESCRIPTION
**Pull request by krokofant**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111766&drop=dogfoodAlpha

### CODE REVIEW

**Comment by krokofant** [comment]
@microsoft-github-policy-service agree

File: manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: wahibre.mtn
+PackageVersion: 3.4.2
+Installers:
+- InstallerUrl: https://bitbucket.org/wahibre/mtn/downloads/mtn-3.4.2-win64.zip
```
Comment (stephengillie) [inline manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: Is this the author's official site? It requires an Atlassian login.

**Comment by stephengillie** [comment]
Note to self: This package installs and launches normally.

File: manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml (hunk 1 above)
Comment (krokofant) [reply manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: @stephengillie This is the url to the project, it doesn't require a login https://bitbucket.org/wahibre/mtn/src/master/

The repository is mirrored to gitlab as well where the nicer documentation is. The binaries are published on the bitbucket mirror.
https://gitlab.com/movie_thumbnailer/mtn/-/wikis/home

Comment (stephengillie) [reply manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: The BitBucket repo claims to be a mirror of the GitLab repo:
image

Comment (krokofant) [reply manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: The downloads from the main _gitlab repo_ are pointed to bitbucket. If you look at https://gitlab.com/movie_thumbnailer/mtn/-/wikis/home, and then navigate to the [downloads](https://gitlab.com/movie_thumbnailer/mtn/-/releases) the direct linux and windows binaries are pointing to _bitbucket_.

I'm not sure what I could do about this, or what info you want me to provide. 😕

Comment (krokofant) [reply manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: Both repos are from wahibre

Comment (stephengillie) [reply manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml:7]: Thank you for explaining this ISV's atypical hosting configuration.

File: manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: wahibre.mtn
+PackageVersion: 3.4.2
+PackageLocale: en-US
+Publisher: wahibre
+PackageName: mtn
```
Comment (stephengillie) [inline manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://bitbucket.org/wahibre/mtn/src/master/
PackageName: mtn
PackageUrl: https://gitlab.com/movie_thumbnailer/mtn/-/wikis/home
```

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**